openslides_backend/action/relations/relation_manager.py

```diff
@@ -163,13 +162,18 @@ def set_structured_field(
         # process template fields and set the contained structured fields
         for field_name, field in template_fields:
             field_value = instance[field_name]
-            assert isinstance(field_value, dict)
-
-            additional_instance_fields[field_name] = get_template_field_db_value(
-                field_name
-            )
-            for replacement, value in field_value.items():
-                set_structured_field(field, str(replacement), value)
+            if isinstance(field_value, dict):
+                additional_instance_fields[field_name] = get_template_field_db_value(
+                    field_name
+                )
+                for replacement, value in field_value.items():
+                    set_structured_field(field, str(replacement), value)
+            elif isinstance(field_value, list):
+                pass  # Todo: check this one: for default_projector_$_id there is the list of possible replacements
```

Comment:
I think this should never happen - do you have a case where a list gets passed here?

Reply:
> I think this should never happen - do you have a case where a list gets passed here?

Yes, for default_projector_$_id there is the list of possible replacements